fuzz/Cargo.toml

```diff
@@ -0,0 +1,65 @@
+[package]
+name = "libbzip2-rs-fuzz"
+version = "0.0.0"
+authors = ["Automatically generated"]
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[profile.release]
+debug = true
+
+[features]
+default = ["rust-allocator", "libbzip2-rs-sys/testing-prefix"]
```

Comment:
I think `testing-prefix` needs to always be enabled, otherwise we're actually just fuzzing the C implemenation (at least, it won out before when both rust and c exposed the same symbols)

Reply:
Enabled it unconditionally.